jobs/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for jobs.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Image for container
+image: hmctssandbox.azurecr.io/hmcts/plum-batch:latest
```

Comment:
do we need a default image?

Reply:
I think , it is fine to have a sensible default ? Happy to get rid of it , but just wanted it to work on helm install

Reply:
not sure you can have a sensible default here though, we've seen people mess this up and get plum, I don't think it really helps someone using that chart to get plum?